src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -104,6 +155,15 @@ class PaymentInformationEditModal extends React.Component {
     });
   };
 
+  formSubmit = ({ formData }) => {
+    this.props.onSubmit({
```

Comment:
This call is pulled out of the component's `onSubmit` method that the current form uses for validation and submission.